### PR TITLE
[luci] Fix unreachable loop in ExpandBroadcastConstPass

### PR DESCRIPTION
**Pull request by llFreetimell**
From second loop, current code is not reachable
because the function is always returned at first iteration.
This commit will fix it.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

**Comment by ai-moiseev** [comment]
In initial pass PR there was a request to follow the style of other passes here: https://github.com/Samsung/ONE/pull/7603#discussion_r702479369... Maybe we can just remove `return false` at the end of the function? :)

**Comment by llFreetimell** [comment]
Using `if()` instead of `= ||` is fine.
But I want to ask which one was your intention :)
Only applying to first `succ` of the node or applying to all of `succ` of the node?

**Comment by ai-moiseev** [comment]
@llFreetimell Oh, it seems it expands only the first successor input, the intention was to expand all successors. Thanks for noticing this mistake :)

File: compiler/luci/pass/src/ExpandBroadcastConstPass.cpp [hunk 1]
```diff
@@ -125,22 +125,25 @@ bool expand_broadcast_const(luci::CircleConst *node)
...
+  bool changed = false;
+
   for (auto successor : loco::succs(node))
   {
     auto const circle_successor = loco::must_cast<luci::CircleNode *>(successor);
     switch (circle_successor->opcode())
     {
       case luci::CircleOpcode::ADD:
-        return expand_node_input<luci::CircleAdd>(node, circle_successor);
+        changed = changed || expand_node_input<luci::CircleAdd>(node, circle_successor);
```
Comment (ai-moiseev) [inline compiler/luci/pass/src/ExpandBroadcastConstPass.cpp:136]: Missing `break` in cases?

**Review by ai-moiseev — APPROVED** [review]
LGTM👍

**Pull request merged**